dev-site-config/navigation.config.js

```diff
@@ -6,9 +6,21 @@ const navConfig = {
       text: 'Home',
       pageTypes: ['home'],
     }, {
-      path: '/getting-started',
-      text: 'Getting Started',
+      path: '/about',
+      text: 'About',
+      pageTypes: ['about'],
+    }, {
+    }, {
+      path: '/guides',
+      text: 'Guides',
       pageTypes: ['guide'],
+    }, {
+      path: '/application',
+      text: 'Application',
```

Comment:
As of now, I can't think of a better name, but Application as one word doesn't feel like it provides enough info. I'll think about this, but we've describe terra-ui as a component library, which terra-app technically is but this is providing the application framework so its more than a component

Reply:
terra-ui is more than a component library (as of a couple of days ago 😄 ) I've tried to address this in the about terra section. I see terra as having three tiers:

Application - Application framework to create applications
Components - ui components to use when composing views for applications
dev environment - webpack configs, testing frameworks terra-dev-site

Reply:
I agree. Maybe the real need is to beef up the Home page to describe these tiers:
https://terra-ui-dep-proposed-r-0pjke2.herokuapp.com/home/terra-ui/index

this mentions about and components

Reply:
Would Internationalization be more fitting under dev env then? It feels like there's overlap between Guides and Dev Environment. I'd be curious what others think make sense conceptually if they were looking at terra-ui as if they were a consumer

Reply:
I think there is currently some overlap between the Guides and  the Application, Components, and Dev ecosystem sections. How I've been looking at it is is that guides contains 'how to' documentation and the other sections contains to the point API doc. I know that a common complaint I have with doc sites is that when i Just want to know what a property does I don't want to have to read a whole 'how to guide' to find it. (I'm looking at you rails docs)